snippets/performance_ncbi_bulk_vs_delta_loader.md

```diff
@@ -14,7 +14,7 @@ continued support.
   exclusively use the delta loader.
   * They are mostly orders of magnitude smaller than the NCBI dataset, in which case loading with
     the delta loader is fast.
-* As of this writing the NCBI delta loader doesn't support release timestamps or registering
+* As of this writing the NCBI bulk loader doesn't support release timestamps or registering
```

Comment:
Why this change? Or was it just previously wrong?

Reply:
It was wrong, it's the bulk loader that doesn't support those features. The delta loader does